controllers/paddlejob_controller.go

```diff
@@ -180,6 +205,12 @@ func (r *PaddleJobReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 			return false
 		}
 		pod := constructPod(&pdj, resType, idx)
+
+		if r.Scheduling == schedulerNameVolcano && !withoutVolcano(&pdj) {
+			pod.ObjectMeta.Annotations[schedulingPodGroupAnnotation] = pdj.Name
```

Comment:
one last suggestion: please add full list annotations of volcano.

ref:
* https://github.com/volcano-sh/volcano/blob/b82b9667a5dd1eca2983f1645cd014cdb43b027c/pkg/controllers/job/job_controller_util.go#L99

Reply:
update